### ISSUE TITLE
How to build the new version?

### ISSUE THREAD

**Issue by pepa65** [PhotoFilmStrip/PFS#19]
If I do `python3 ./setup.py build` I get `Cannot update image resources! Using images.py from source`

**Comment by jensgoe** [comment]
this should be only a warning, but if you want to get rid of this you have to install wxPython (pip install wxpython). This you need anyway to run the app.

**Comment by pepa65** [comment]
PFS used to build so easily! What happened?? I tried installing wxPython but ran into this:

`sudo pip3 install wxpython`:
```
    checking for GTK+ - version >= 3.0.0... Package gtk+-3.0 was not found in the pkg-config search path.
    Perhaps you should add the directory containing `gtk+-3.0.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'gtk+-3.0' found
    no
    *** Could not run GTK+ test program, checking why...
    *** The test program failed to compile or link. See the file config.log for the
    *** exact error that occured. This usually means GTK+ is incorrectly installed.
    configure: error:
    The development files for GTK+ were not found. For GTK+ 2, please
    ensure that pkg-config is in the path and that gtk+-2.0.pc is
    installed. For GTK+ 1.2 please check that gtk-config is in the path,
    and that the version is 1.2.3 or above. Also check that the
    libraries returned by 'pkg-config gtk+-2.0 --libs' or 'gtk-config
    --libs' are in the LD_LIBRARY_PATH or equivalent.
    
    Error running configure
    ERROR: failed building wxWidgets
    Traceback (most recent call last):
      File "build.py", line 1295, in cmd_build_wx
        wxbuild.main(wxDir(), build_options)
      File "/tmp/pip-build-s87772h9/wxpython/buildtools/build_wxwidgets.py", line 374, in main
        "Error running configure")
      File "/tmp/pip-build-s87772h9/wxpython/buildtools/build_wxwidgets.py", line 85, in exitIfError
        raise builder.BuildError(msg)
    buildtools.builder.BuildError: Error running configure
    Finished command: build_wx (0m11.172s)
    Finished command: build (0m11.172s)
    Command '"/usr/bin/python3" -u build.py build' failed with exit code 1.
    
    ----------------------------------------
Command "/usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-s87772h9/wxpython/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-247c20ns-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-s87772h9/wxpython/
```
I have `libgtk-3-0` installed, it seems Ubuntu 16.04 doesn't have much else.

**Comment by jensgoe** [comment]
you probably need the dev-package libgtk-3-0-dev. Have a look at https://github.com/wxWidgets/Phoenix/ there you will finad a list of packagas that are needed to bulid wxPython4

**Comment by pepa65** [comment]
Hmm they only have libgtk-3-0 and libgtk-3-0-dbg. I am now running version 3.3.1 which still works. I'll wait until the situation improves.

**Comment by jensgoe** [comment]
Thats right. wxPython4 is only necessary since PhotoFilmStrip version 3.4.0. Version 3.3.1 still works with wxPython3 which is in the official packages. wxPython4 is in progress as far as i knowl.

**Comment by debalance** [comment]
> I have libgtk-3-0 installed, it seems Ubuntu 16.04 doesn't have much else.

The name of the -dev package in Debian+Ubuntu is "libgtk-3-dev":
https://packages.ubuntu.com/xenial/libgtk-3-dev

**Comment by pepa65** [comment]
@debalance Thanks, that is good to know. I'll try again.

**Comment by pepa65** [comment]
Alright, got  it all to build after getting wxpython4. Besides `libgtk-3-dev` I installed `libwebkit2gtk-4.0-dev` (probably not necessary) `libgstreamer1.0-dev` (not enough on its own) `libgstreamer-plugins-base1.0-dev libgstreamer-plugins-bad1.0-dev libgstreamer-plugins-good1.0-dev` (possibly not all needed). Then `sudo pip3 install wxpython` succeeded. And `python3 setup.py build` plus `sudo python3 setup.py install` eventually gave a working install.

The only trouble was remnants of old installs... I had already purged the repo's version of photofilmstrip with the package manager. But I found the install of 3.3.1 (and possibly pear-shaped attempts of 3.4.1) needed to be deleted as well, I did `sudo rm -rf /usr/local/bin/photofilmstrip* /usr/bin/photofilmstrip* /usr/local/lib/*/dist-packages/photofilmstrip`. It is too bad that the `setup.py` doesn't provide a way to cleanly uninstall...
